lib/hypertypes/hypertypes/ERC20MintableContract.py

```diff
@@ -66,11 +66,11 @@ class ERC20MintableBurnContractFunction(ContractFunction):
 
     def __call__(self, amount: int, destination: str | None = None) -> "ERC20MintableBurnContractFunction":
         if all([destination is None]):
-            super().__call__(amount)
+            super().__call__()
```

Comment:
these two conditions are the same now, so why have the conditional at all?

Reply:
this actually looks like a bug